package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "yearn-lens",
+  "devDependencies": {
+    "@commitlint/cli": "^11.0.0",
+    "@commitlint/config-conventional": "^11.0.0",
+    "ethlint": "^1.2.5",
```

Comment:
Will we use both this and prettier?

Reply:
Oh no, i've copied package.json from brownie-strategy-mix, will remove it now, thank you.